Common/tests/TestBase/AuthenticationTableTestBase.cs

```diff
@@ -38,6 +39,16 @@ public class AuthenticationTableTestBase
   [SetUp]
   public async Task SetUp()
   {
+    Environment.SetEnvironmentVariable(CertificatePath.Replace(":",
+                                                               "__") + "__0",
+                                       CertEnv.ToJson());
+    Environment.SetEnvironmentVariable(UserPath.Replace(":",
+                                                        "__") + "__0",
+                                       UserEnv.ToJson());
+    Environment.SetEnvironmentVariable(RolePath.Replace(":",
+                                                        "__") + "__0",
+                                       RoleEnv.ToJson());
```

Comment:
Can't you have RolePath directly into the right format?

Reply:
I also use it in the  config map that is directly given to the ConfigurationBuilder